src/tests/Microsoft.PowerFx.Core.Tests/DPathTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Core.Utils;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class DPathTest 
```

Comment:
>DPathTest [](http://example.com/codeflow?start=17&length=9)

Another case (From DPathTEst.cs in Paclient)
        [TestMethod, Owner("hekum")]
        public void TryParseEmptyString()
        {
            DPath result;
            Assert.IsTrue(DPath.TryParse("", out result));
            Assert.AreEqual(DPath.Root, result);
        }

Reply:
Although I see we're removing TryParse, so maybe this test doesn't matter.

Reply:
Yeah, deliberately removed. See corresponding PR in Client Repo.

Reply:
ok - there's still that other question about GoUp